types/jsdom/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for jsdom 11.12
+// Type definitions for jsdom 12.2
 // Project: https://github.com/tmpvar/jsdom#readme
 // Definitions by: Leonard Thieu <https://github.com/leonard-thieu>
 //                 Johan Palmfjord <https://github.com/palmfjord>
```

Comment:
can add myself here if you guys @leonard-thieu & @palmfjord don't want to own the ResourceLoader part

Reply:
Sure, feel free!